datastream-server-restli/src/main/java/com/linkedin/datastream/server/dms/DatastreamStore.java

```diff
@@ -72,4 +73,19 @@ void updatePartitionAssignments(String key, Datastream datastream, HostTargetAss
    * @param key Name of the datastream whose assignment tokens have to be deleted
    */
   void forceCleanupDatastream(String key);
+
+  /**
+   * Creates or Updates the datastream entry in the persistent store with the latest list of violating topics.
```

Comment:
`persistent store` is an implementation specifics.  Please make it generic for the interface class.